source/meta.tex

```diff
@@ -1589,6 +1607,26 @@
  \tcode{Fn}, \tcode{R}, and all types in the template parameter pack \tcode{ArgTypes}
  shall be complete types, \cv{}~\keyword{void}, or
  arrays of unknown bound.                                             \\
+
+\indexlibraryglobal{is_applicable}%
+\tcode{template<class Fn, class Tuple>}\br
+ \tcode{struct is_applicable;}              &
+ \exposconcept{tuple-like} is \tcode{true} and
+ the expression \tcode{\placeholdernc{INVOKE}(declval<Fn>(), \placeholdernc{ELEMS-OF}(Tuple)...)}
+ is well-formed when treated as an unevaluated operand.       &
+ \tcode{Fn} and \tcode{Tuple}
+ shall be complete types, \cv{}~\keyword{void}, or
+ arrays of unknown bound.                                             \\ \rowsep
+
+\indexlibraryglobal{is_nothrow_applicable}%
+\tcode{template<class Fn, class Tuple>}\br
+ \tcode{struct is_nothrow_applicable;}              &
+ \tcode{is_applicable_v<}\br\tcode{Fn, Tuple>} is \tcode{true} and
+ the expression \tcode{\placeholdernc{INVOKE}(declval<Fn>(), \placeholdernc{ELEMS-OF}(Tuple)...)}
+ is known not to throw any exceptions\iref{expr.unary.noexcept}       &
```

Comment:
```suggestion
 is known not to throw any exceptions\iref{expr.unary.noexcept}.       &
```

Reply:
Well, I was just going off of paper discrepancies. There is a period in the paper at that point. I would assume that there should be a fixup commit for consistency then.

Reply:
Fixed, and #8005 opened for the inconsistency.